fbchat/_client.py

```diff
@@ -2925,10 +2925,6 @@ def onLoggingIn(self, email=None):
         """
         log.info("Logging in {}...".format(email))
 
-    def on2FACode(self):
```

Comment:
@madsmtm Should we remove this or rather leave it as it is? This function won't be called anymore, so I think we can just do this small breaking change.